doc/Support/SNMP-Configuration-Examples.md

```diff
@@ -0,0 +1,150 @@
+source: Support/SNMP-Configuration-Examples.md
+
+# SNMP configuration examples
+
+Table of Content:
+- [Devices](#devices)
+    - [Cisco](#cisco)
+        - [Adaptive Security Appliance (ASA)](#adaptive-security-appliance-asa)
+        - [IOS](#ios)
+        - [IOS](#ios)
+        - [IOS XE](#ios-xe)
+        - [NX-OS](#nx-os)
+        - [Wireless LAN Controller (WLC)](#wireless-lan-controller-wlc)
+    - [Infoblox](#infoblox)
+        - [NIOS 7.x](#nios-7x)
+    - [Juniper](#juniper)
+        - [Junos OS](#junos-os)
+    - [Palo Alto](#palo-alto)
+        - [PANOS 6.x/7.x](#panos-6x7x)
+- [Operating systems](#operating-systems)
+    - [Linux (snmpd)](#linux-snmpd)
+    - [Windows Server 2012 R2](#windows-server-2012-r2)
+
+## Devices
+
+### Cisco
+#### Adaptive Security Appliance (ASA)
+1. Launch ASDM and connect to your device
+2. Go to Configuration > Management Access > SNMP
+3. Add your community string
+4. Add in the "SNMP Host Access List" section your LibreNMS server IP address
+5. Click Apply and Save
+
+#### IOS
```

Comment:
Might as well combine all the IOS ones, I don't see any reason they would need to be separate.